osu.Game/Screens/Multiplayer/DrawableRoom.cs

```diff
@@ -59,24 +68,36 @@ public DrawableRoom(Room room)
                 new Box
                 {
                     RelativeSizeAxes = Axes.Both,
-                    Colour = OsuColour.Gray(34),
+                    Colour = OsuColour.FromHex(@"212121"),
                 },
                 sideStrip = new Box
                 {
                     RelativeSizeAxes = Axes.Y,
-                    Width = content_padding,
+                    Width = side_strip_width,
                 },
-                avatar = new UpdateableAvatar
+                new Container
                 {
-                    Size = new Vector2(Height - content_padding * 2),
+                    Width = cover_width,
+                    RelativeSizeAxes = Axes.Y,
                     Masking = true,
-                    CornerRadius = 5f,
-                    Margin = new MarginPadding { Left = content_padding * 2, Top = content_padding },
+                    Margin = new MarginPadding { Left = side_strip_width },
+                    Children = new Drawable[]
+                    {
+                        new Box
+                        {
+                            RelativeSizeAxes = Axes.Both,
+                            Colour = Color4.Black,
+                        },
+                        coverContainer = new Container
+                        {
+                            RelativeSizeAxes = Axes.Both,
+                        },
+                    },
                 },
                 new Container
                 {
                     RelativeSizeAxes = Axes.Both,
-                    Padding = new MarginPadding { Top = content_padding, Bottom = content_padding, Left = Height + content_padding * 2, Right = content_padding },
+                    Padding = new MarginPadding { Top = content_padding, Bottom = content_padding, Left = side_strip_width + cover_width + content_padding, Right = content_padding },

```

Comment:
i still think cases like this should be split into newlines per property setter.